modules/javafx.controls/src/main/java/javafx/scene/control/TabPane.java

```diff
@@ -816,16 +735,16 @@ private Tab findNearestAvailableTab(int tabIndex, boolean doSelect) {
     }
 
     /**
-     * <p>This specifies how the TabPane handles tab closing from an end-users
+     * <p>This specifies how the {@code TabPane} handles tab closing from an end-user's
```

Comment:
Since you are fixing the description here, can you remove the "This" in the beginning?

Reply:
Good suggestion.

Also optional, you can remove the `<p>` since they are not needed for the initial paragraph.

Reply:
There are many redundant `<p>` tags in the property docs too if you want to deal with them as well for consistency (not important).

Reply:
Fixed the description. Also, removed many redundant <p> tags.